websites/serializers.py

```diff
@@ -209,6 +217,50 @@ def update(self, instance, validated_data):
         update_website_backend(instance.website)
         return instance
 
+    def get_content_context(self, instance):
+        """
+        Create mapping of uuid to a display name for any values in the metadata
+        """
+        if self.context and self.context.get("no_content_context"):
+            # Prevent circular dependencies where two relation fields may refer to one another
+            return None
+
+        text_ids = []
+        website_name = None
+        metadata = instance.metadata or {}
+        site_config = SiteConfig(instance.website.starter.config)
+        for field in site_config.iter_fields():
+            if field.field.get("widget") == "relation":
+                try:
```

Comment:
Can we add a relation widget to the test fields?  This whole block currently isn't hit.

Reply:
It looks like this is covered by tests. If I add a `raise Exception()` at this line the `test_website_content_detail_serializer_content_context` test fails in certain conditions

Reply:
It seems like for this and the above coverage issues I mentioned, the Codecov data was out of date and it appeared that these cases weren't covered.  Either way, it looks good now.